api/ocm/extensions/repositories/genericocireg/accessmethod_localblob.go

```diff
@@ -88,7 +91,17 @@ func (m *localBlobAccessMethod) getBlob() (blobaccess.DataAccess, error) {
 			return nil, errors.ErrNotImplemented("artifact blob synthesis")
 		}
 	}
-	_, data, err := m.namespace.GetBlobData(digest.Digest(m.spec.LocalReference))
+	refs := strings.Split(m.spec.LocalReference, ",")
+
+	var (
+		data blobaccess.DataAccess
+		err  error
+	)
+	if len(refs) < 2 {
+		_, data, err = m.namespace.GetBlobData(digest.Digest(m.spec.LocalReference))
+	} else {
+		data = &composedBlock{m, refs}
+	}
```

Comment:
why not handle the error locally within the if below line 101?

Reply:
Because the else was different before :-),  fixed...